app/services/magic_link_sign_up_flow.rb

```diff
@@ -0,0 +1,25 @@
+class MagicLinkSignUpFlow
+  def self.begin(candidate)
+    magic_link_token = MagicLinkToken.new
+    candidate.update!(magic_link_token: magic_link_token.encrypted)
```

Comment:
Why do you need to update twice the candidate?
Can you update the candidate just once with both `magic_link_token` and `magic_link_token_sent_at` params?

Reply:
Good point! We've updated it now so that there's only a single update after the email is delivered.